templates/events/details.html

```diff
@@ -393,6 +418,54 @@ <h3>Meld deg av {{ event.title }} </h3>
                 </div>
             </div>
 
+            <div id="attendees" class="modal fade">
+                <div class="modal-dialog">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                            <h3>Påmeldte til {{ event.title }} </h3>
+                            <h5><i>Siden dette er et sosialt arrangement kan du ønske å vise deg som påmeldt for andre å se.</i></h5>
+                        </div>
+                        <div class="modal-body">
+                            <table class="table table-striped table-condensed tablesorter attendees" id="extras-table">
+                                <thead>
+                                    <tr>
+                                        <th><b>Fornavn</b></th>
+                                        <th><b>Etternavn</b></th>
+                                    </tr>
+                                </thead>
+                                <tbody id="attendee_list">
+                                    {% for user in event.attendance_event.visible_attendees_qs %}
+                                    <tr>
+                                        <td>{{ user.first_name }}</td>
+                                        <td>{{ user.last_name }}</td>
+                                    </tr>
+                                    {% endfor %}
+                                </tbody>
+                            </table>
+                             {% if user_setting_show_as_attending %}
+                            <span><i>Du vises automatisk som påmeldt for alle sosiale arrangement, dette kan endres under personvern på <a href='/profile/privacy'>profilen din her.</a></i></span>
+                            {% else %}
+                            <span><i>Hvis du ønsker å automatisk vises som påmeldt på alle fremtidige sosiale arrangement kan det gjøres under personvern på <a href='/profile/privacy'>profilen din her.</a></i></span>
```

Comment:
Use the Django URL template tags for URLs, like `{% url 'profile_index' %}` or whatever the name is (check urls.py for profile).